.github/workflows/project-automation.yml

```
@@ -0,0 +1,78 @@
+                      value: { singleSelectOptionId: "${commitStatus}" }
+                    }
+                  ) {
+                    projectV2Item {
+                      id
+                    }
+                  }
+                }
+              `);
+            } catch (error) {
+              console.error('Error updating project:', error);
+            }
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)